src/index.d.ts

```diff
@@ -1,9 +1,10 @@
 // Type definitions for browser-line-reader 0.02
 // Project: browser-line-reader
 // Definitions by: Stanley Clark <me@stanrogo.com>
+import {Options} from './interfaces';
```

Comment:
This and a few other lines are not consistent with the linting rules set out in pritterrc and eslintrc. That's on me that I didn't include proper contribution guidelines (and the pre-commit hooks didn't seem to work). For now, please just accept my suggestions and I'll look into this at some point.